.github/workflows/CI.yaml

```
@@ -0,0 +1,43 @@
+name: Event Connect - CI
+on: [pull_request]
+jobs:
+  event-connect-build:
+    env:
```

Please add [`concurrency` element](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#concurrency) to cancel old jobs when pushing new code
```suggestion
    concurrency: 
      group: ${{ github.head_ref }}
      cancel-in-progress: true
    env:
```

---

concurrency element already present